update-data.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+unicode_version="12.0.0"
```

Comment:
Huh? The latest version is v12.1.0.

Reply:
Oh oops. TBH I didn’t even check. Just had 12 in my mind I guess. 😂